packages/build/bin/utils.js

```diff
@@ -214,9 +214,18 @@ function isOptionSet(opts, ...optionNames) {
   );
 }
 
-function mochaOptsFileProjectExists() {
-  const configFile = path.join(getPackageDir(), 'test/mocha.opts');
-  return fs.existsSync(configFile);
+function mochaConfiguredForProject() {
+  const configFiles = [
+    '.mocharc.js',
+    '.mocharc.json',
+    '.mocharc.yaml',
+    '.mocharc.yml',
+    'mocha.opts',
```

Comment:
I believe the default config file is `test/mocha.opts`, not `mocha.opts`. Has this changed in `mocha@6`?

Reply:
Good catch. I fixed it.